src/sentry/api/endpoints/organization_events_histogram.py

```diff
@@ -79,11 +79,11 @@ def get(self, request: Request, organization) -> Response:
 
                 with handle_query_errors():
                     results = dataset.histogram_query(
-                        data["field"],
-                        data.get("query"),
-                        params,
-                        data["numBuckets"],
-                        data["precision"],
+                        fields=data["field"],
+                        user_query=data.get("query"),
+                        params=params,
+                        num_buckets=data["numBuckets"],
+                        precision=data["precision"],
```

Comment:
Didn't really want to fiddle with this more than I needed to, just added all the args instead of reordering it